docs/content/latest/deploy/docker/docker-swarm.md

```diff
@@ -227,7 +230,8 @@ $ docker service create \
 --publish 9000:9000 \
 yugabytedb/yugabyte:latest /home/yugabyte/bin/yb-tserver \
 --fs_data_dirs=/mnt/data0 \
---tserver_master_addrs=yb-master1:7100,yb-master2:7100,yb-master3:7100
+--rpc_bind_addresses=0.0.0.0:9100 \
```

Comment:
@iSignal Does the yb-tserver setting also need to be changed?

Reply:
yes, it does!

Reply:
No, this change isn't required. For every "yb-master" we execute its own create' command that's why we can give specific hostname or service name whereas in the case of "yb-tserver" we create multiple instances with a single command, therefore, giving hostname or service name is not possible in case of "yb-tserver".